Cargo.toml

```diff
@@ -93,6 +81,7 @@ url.workspace = true
 wait-timeout = "0.2"
 xz2 = "0.1.3"
 zstd = "0.12"
+atty = "0.2.14"
```

Comment:
Please sort this (see my suggestion about IDE support up above - though finding a third party CLI tool to do it would be nice too).